support-frontend/webpack.entryPoints.js

```diff
@@ -33,6 +33,6 @@ module.exports = {
 		ausMomentMap: 'pages/aus-moment-map/ausMomentMap.tsx',
 	},
 	ssr: {
-		ssrPages: 'helpers/rendering/ssrPages.ts',
+		ssrPages: 'scripts/build-ssr-content.tsx',
```

Comment:
Presume this replaces old ssrPages.ts with a version that can display either `NewPaperArchive` or `HoldingContent`?

Reply:
Yep, not sure if it should be a webpack entrypoint. The only purpose it serves here is to be an exception for the orphans script

Reply:
We don't need this